quick-start.md

```diff
@@ -2,222 +2,87 @@
 
 The average time for Kontena users to get from zero to hero is just under 20 minutes! Let's assume you have already created some amazing application and packaged it as docker containers, here's what it takes to get from zero to hero with Kontena:
 
-1. Install Kontena CLI tool.
-2. Create a [Kontena Platform](using-kontena/platform.md).
-3. Install [Kontena Nodes](using-kontena/nodes.md).
-4. Deploy first [Kontena Stack](using-kontena/stacks.md).
+1. Sign-up to Kontena Cloud.
+2. Create an organization for Kontena Platforms.
+3. Create a [Kontena Platform](using-kontena/platform.md).
+4. Install [Kontena Nodes](using-kontena/nodes.md).
+5. Deploy first [Kontena Stack](using-kontena/stacks.md).
 
 Follow these steps to get started with Kontena on Kontena Cloud quickly. If you like to tinker and maintain the Kontena Platform yourself, please follow the [slow start](./slow-start.md) guide.
```

Comment:
Follow these steps to get started with Kontena on Kontena Cloud quickly. If you'd like to tinker and maintain the Kontena Platform yourself, please follow the [slow start](./slow-start.md) guide.